<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>495</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>423</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>423</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TofConverter</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>401</width>
        <height>422</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input value:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="inputWorkspace"/>
       </item>
       <item>
        <widget class="QLineEdit" name="InputVal">
         <property name="toolTip">
          <string>The value that you wish to convert</string>
         </property>
         <property name="whatsThis">
          <string>An input box for entering the value that you wish to convert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="inputUnits">
         <property name="toolTip">
          <string>A list of the units for the input value</string>
         </property>
         <property name="whatsThis">
          <string>Drop down menu of the units for the input value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Scattering Angle (Degrees):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scatteringAngleInput">
         <property name="toolTip">
          <string>The angle through which a scattered particle or beam is deflected in degrees</string>
         </property>
         <property name="whatsThis">
          <string>An input box for entering the scattering angle in Degrees (if necessary)</string>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Total flight path (Meters):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalFlightPathInput">
         <property name="toolTip">
          <string>The distance travelled by the object measured in meters</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>An input box for entering the total flight path in meters (if necessary)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Convert To:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="outputUnits">
         <property name="toolTip">
          <string>The units that you wish to convert your input value to.</string>
         </property>
         <property name="whatsThis">
          <string>Drop down menu of the units for the input value to be converted to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="convertButton">
         <property name="whatsThis">
          <string>A button to produce the converted value</string>
         </property>
         <property name="text">
          <string>Convert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Converted value:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="convertedVal">
         <property name="toolTip">
          <string>The converted value produced</string>
         </property>
         <property name="whatsThis">
          <string>A output box for displaying the converted value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Help</string>
         </property>
         <property name="whatsThis">
          <string>Help button</string>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
